src/index.ts

```diff
@@ -279,7 +280,27 @@ export default runExtension(async ({ extensionAPI }) => {
             })(...args);
             return result;
           } catch (e) {
-            console.error(e);
+            const error = e as Error;
+            console.error(error);
+            apiPost({
+              domain: "https://api.samepage.network",
+              path: "errors",
+              data: {
+                method: "extension-error",
+                type: "Custom Command Failed",
+                data: {
+                  text: command,
+                },
+                message: error.message,
+                stack: error.stack,
+                version: process.env.VERSION,
+                notebookUuid: JSON.stringify({
+                  owner: "RoamJS",
+                  app: "smartblocks",
+                  workspace: window.roamAlphaAPI.graph.name,
+                }),
+              },
+            }).catch(() => {});
```

Comment:
The email in the screenshot likely didn't come from a custom command bc we are catching the errors and just returning a string here